packages/oui-header-tabs/README.md

```diff
@@ -0,0 +1,68 @@
+# Header tabs
+
+<component-status cx-design="complete" ux="rc"></component-status>
+
+## Usage
+
+### Normal
+
+```html:preview
+<div class="oui-doc-preview-only-keep-children">
```

Comment:
What's the purpose of this `<div>` ?
It seems useless, since it has no style or init.